app/models/user.rb

```diff
@@ -10,6 +10,9 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :trackable, :validatable,
          :confirmable, :recoverable# , :omniauthable
 
+  validates :username, presence: true
+  validates :username, uniqueness: true, if: -> { self.username.present? }
```

Comment:
We will use username only in emails and profile page for now. Don't want to push him create uniq one.
So, let's skip uniq validation for now